selfservice/flow/flow.go

```diff
@@ -38,3 +41,18 @@ type Flow interface {
 	AppendTo(*url.URL) *url.URL
 	GetUI() *container.Container
 }
+
+func IsWebViewFlow(ctx context.Context, conf *config.Config, f Flow) (bool, error) {
```

Comment:
From a security perspective, any client can become a WebView by adding the correct `return_to` parameter, even a malicious JS script.